CMakeLists.txt

```diff
@@ -243,6 +243,14 @@ endif()
 option(UA_ENABLE_PUBSUB "Enable publish/subscribe" OFF)
 mark_as_advanced(UA_ENABLE_PUBSUB)
 
+option(UA_ENABLE_PUBSUB_CONFIG "Enable loading PubSub Config from file extension" OFF)
+mark_as_advanced(UA_ENABLE_PUBSUB_CONFIG) 
+if(UA_ENABLE_PUBSUB_CONFIG)
+    if(NOT UA_ENABLE_PUBSUB)
+        message(FATAL_ERROR "PubSub needs to be enabled")
+    endif()
+endif()
+
```

Comment:
The order and format of the pubsub flags has changed in the last weeks. It would be nice, if you can make a rebase and put your flag in the 'pubsub flag list'.